raft/testdata/snapshot_succeed_via_app_resp.txt

```diff
@@ -0,0 +1,142 @@
+# TestSnapshotSucceedViaAppResp regression tests the situation in which a snap-
+# shot is sent to a follower at the most recent index (i.e. the snapshot index
+# is the leader's last index is the committed index). In that situation, a bug
+# in the past left the follower in probing status until the next log entry was
+# committed.
```

Comment:
If you're using this as a form of a regression test, could you link to the issue/PR you're talking about. Also, make it clear exactly which step this was previously broken in. Is that "Note how it reacts with a MsgAppResp upon completion."?

Reply:
Done.